systemtest/tests/integration/test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -eu
+set -x
+
+. /etc/os-release 
+VERSION_MAJOR=$(echo $VERSION_ID | cut -d. -f 1)
+case "${ID}" in
+  rhel)
+    ;;
+  *)
+    curl --output-dir /etc/yum.repos.d/ -O \
+      "https://ftp.redhat.com/redhat/client-tools/client-tools-for-rhel-${VERSION_MAJOR}.repo"
+    curl -o /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release \
+      https://www.redhat.com/security/data/fd431d51.txt
+    ;;
+esac
+
+# get to project root
+cd ../../../
+
+dnf --setopt install_weak_deps=False install -y \
+  subscription-manager \
+  insights-client \
+  podman git-core python3-pip python3-pytest
+
```

Comment:
would also be nice to be able to run this locally using a tmt container; eg: `tmt run --all provision --how=container --image=fedora:latest`

we would need to add the dbus-devel package and this code for this: 
```
if [ ! `command -v rhc` ]; then
  make install
fi
```
That also means we would have to massage the subman commands in the test plugin (since we'll get container mode warnings).  @ptoscano what would your thoughts be on that?  I think it would be nice for developing these tests.

Reply:
I have removed installing python packages directly in test.sh. They will be installed using requirements.txt instead.